dietpi/dietpi-software

```diff
@@ -8125,7 +8132,26 @@ _EOF_
 			G_EXEC_NOHALT=1 G_EXEC rm install.bash
 
 		fi
+		
+                software_id=203 # Zerotier
+                if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+                        Banner_Installing
+			 
+			#Purge down in case of reinstall
+                        [[ -f '/usr/sbin/zerotier-cli' ]] && G_EXEC rm /usr/sbin/zerotier-cli
+                        [[ -f '/usr/sbin/zerotier-one' ]] && G_EXEC rm /usr/sbin/zerotier-one
+                        [[ -f '/etc/apt/sources.list.d/zerotier.list' ]] && G_EXEC rm /etc/apt/sources.list.d/zerotier.list
+                        [[ -f '/var/lib/zerotier-one/identity.public' ]] && G_EXEC rm /var/lib/zerotier-one/identity.public
+                        [[ -f '/etc/init.d/zerotier-one' ]] && G_EXEC rm /etc/init.d/zerotier-one
+			G_EXEC apt remove zerotier-one -y
+                        #Installation
+			G_EXEC wget -O zerotier-installer.sh https://install.zerotier.com
```

Comment:
let's try to avoid using the install script. Better to install key and source list on our own. Following could do

			Banner_Installing

			# APT key
			G_EXEC eval "curl -sSLf https://raw.githubusercontent.com/zerotier/ZeroTierOne/master/doc/contact%40zerotier.com.gpg | gpg --dearmor -o /etc/apt/trusted.gpg.d/dietpi-zerotier.gpg --yes"

			# APT list
			G_EXEC eval "echo 'deb https://download.zerotier.com/debian/$G_DISTRO_NAME $G_DISTRO_NAME main' > /etc/apt/sources.list.d/zerotier.list"

			G_AGUP
			G_AGI tailscale
		fi